src/cli-pull-ot.ts

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+import { logger } from './logger';
+import colors from 'colors';
+import {
+  getListOfAssessments,
+  getAssessment,
+  writeOneTrustAssessment,
+  parseCliPullOtArguments,
+  createOneTrustGotInstance,
+} from './oneTrust';
+import { OneTrustPullResource } from './enums';
+import { mapSeries } from 'bluebird';
+
+/**
+ * Pull configuration from OneTrust down locally to disk
+ *
+ * Dev Usage:
+ * yarn ts-node ./src/cli-pull-ot.ts --hostname=customer.my.onetrust.com --auth=$ONE_TRUST_OAUTH_TOKEN --file=./oneTrustAssessment.json
+ *
+ * Standard usage
+ * yarn cli-pull-ot --hostname=customer.my.onetrust.com --auth=$ONE_TRUST_OAUTH_TOKEN --file=./oneTrustAssessment.json
+ */
+async function main(): Promise<void> {
+  const { file, fileFormat, hostname, auth, resource, debug } =
+    parseCliPullOtArguments();
+
+  try {
+    if (resource === OneTrustPullResource.Assessments) {
```

Comment:
nit: best to move these into helper functions so that they can be imported as code if needed

Reply:
will do in a follow up PR!